pyrene/src/components/TreeTable/TreeTable.jsx

```diff
@@ -413,6 +414,10 @@ TreeTable.propTypes = {
    * Whether the columns (hide/show) popover is available to the user.
    */
   toggleColumns: PropTypes.bool,
+  /**
+   * Callback function when the columns of the table are getting toggled.
+   */
+  toggleColumnsCallback: PropTypes.func,
```

Comment:
@sosostris I get your idea. Look fine to me. I would have called the `props` like the following but it does not matter much.

`toggleColumnsHandler`

Reply:
Could also be events/callback style with `onColumnsToggle`, but no strong opinion for any of the options. Maybe actually, the `onColumnsToggle` implies too much the `onChange`/controlled pattern?